crates/isograph_lang_parser/src/parse_iso_literal.rs

```diff
@@ -54,8 +90,8 @@ pub fn parse_iso_fetch(
     Ok(resolver_fetch)
 }
 
-pub fn parse_iso_literal(
-    iso_literal_text: &str,
+fn parse_iso_client_field(
+    tokens: &mut PeekableLexer<'_>,
```

Comment:
Can you make the return type `ParseResultWithLocation<WithSpan<ResolverDeclaration>>` here, since we're no longer using the optional name anywhere?